definitions/divorce/json/CaseTypeTab.json

```diff
@@ -250,957 +250,897 @@
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "CostsClaimGranted",
-    "TabFieldDisplayOrder": 57
+    "TabFieldDisplayOrder": 3
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "WhoPaysCosts",
-    "TabFieldDisplayOrder": 58
+    "TabFieldDisplayOrder": 4
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "TypeCostsDecision",
-    "TabFieldDisplayOrder": 59
+    "TabFieldDisplayOrder": 5
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "CostsOrderAdditionalInfo",
-    "TabFieldDisplayOrder": 60
+    "TabFieldDisplayOrder": 6
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "BulkListingCaseId",
-    "TabFieldDisplayOrder": 58
+    "TabFieldDisplayOrder": 7
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "CourtName",
-    "TabFieldDisplayOrder": 59
+    "TabFieldDisplayOrder": 8
   },
   {
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "DIVORCE",
     "Channel": "CaseWorker",
-    "TabID": "petitionerDetails",
-    "TabLabel": "Petitioner",
+    "TabID": "outcomeOfDnDetails",
+    "TabLabel": "Outcome of Decree Nisi",
     "TabDisplayOrder": 3,
     "CaseFieldID": "DateAndTimeOfHearing",
-    "TabFieldDisplayOrder": 60
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "DIVORCE",
-    "Channel": "CaseWorker",
-    "TabID": "respondentDetails",
-    "TabLabel": "Respondent",
-    "TabDisplayOrder": 4,
-    "CaseFieldID": "D8caseReference",
-    "TabFieldDisplayOrder": 1
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "DIVORCE",
-    "Channel": "CaseWorker",
-    "TabID": "respondentDetails",
-    "TabLabel": "Respondent",
-    "TabDisplayOrder": 4,
-    "CaseFieldID": "createdDate",
-    "TabFieldDisplayOrder": 2
```

Comment:
`D8caseReference` and `createdDate` are petitioner fields so removed them from AOS